src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +286,40 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was deprecated.
+* File handles can now be serialized and stored in IndexedDB, or sent via `postMessage`
+  to other windows or workers within the same origin. Note that permissions
+  are not retained between browser sessions. For example, when a browser tab
+  is re-opened, and a file handle is obtained from IndexedDB, the user will
+  need to grant permission to read/write to the file again.
```

Comment:
it is probably important to mention somewhere that currently chrome won't automatically prompt for read access if you try to do a operation a handle where read permission expired (for example restored from IndexedDB in a new browser session). If `await handle.queryPermission()` returns "prompt" you need to explicitly call `await handle.requestPermission()` before you might be able to use the handle.

Reply:
Added some content to cover that around line 300